systemd/turbinia_config_parser.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't think we want to create another top level directory for this.  Maybe put this in 'tools' for now, and maybe we can refactor that into something that makes more sense?

Reply:
Done